recipes/brainbox/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set name = "brainbox-ibl" %}
+{% set version = "1.0.0" %}
+
+ package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+ source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8968ece4cba16a4b792cb0d06bf714007375b4c9c52c109fec52ceabc1a39d61
+
+ build: 
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
+
+ requirements:
+  host:
+    - python>=3
```

it needs an empty space after the package
```suggestion
    - python >=3
```

---

ah, thanks!